src/store/store.ts

```diff
@@ -95,15 +125,15 @@ const reducer = createReducer(initialState, (builder) => {
         id: state.issues.length,
         title,
         description,
-        status: Status.New,
+        status: "NEW",
```

Comment:
The InitialStatus enum can also be used here

Reply:
done